packages/xdl/src/Project.ts

```diff
@@ -511,51 +510,117 @@ export async function exportForAppHosting(
     await writeArtifactSafelyAsync(
       projectRoot,
       null,
-      path.join(outputDir, 'debug.html'),
+      path.join(absoluteOutputDir, 'debug.html'),
       debugHtml
     );
   }
 
-  const hookOptions = {
-    url: null,
-    exp,
-    iosBundle,
-    iosSourceMap,
-    iosManifest,
-    androidBundle,
-    androidSourceMap,
-    androidManifest,
-    projectRoot,
-    log: (msg: any) => {
-      logger.global.info({ quiet: true }, msg);
-    },
-  };
+  // Skip the hooks and manifest creation if building for EAS.
+  if (!eas) {
```

Comment:
Everything in this `if` statement is copied over from before. Because `if (options.dumpSourcemap)` got shifted above the "hooks" block it looks like more change than it is.